### PR TITLE
Polkadot v0.9.4

### PR DESCRIPTION
**Pull request by Satoshi-Kusumoto**

### CODE REVIEW

**Comment by h4n0** [comment]
It seems only account linker and identity are adapted, but ocw is not. Is it on purpose?

**Review by geastwood — COMMENTED** [review]
Looks consistent.

**Comment by Satoshi-Kusumoto** [comment]
> It seems only account linker and identity are adapted, but ocw is not. Is it on purpose?

ocw is updated

**Review by suinuj — APPROVED** [review]

File: pallets/account-linker/src/lib.rs [hunk 1]
```diff
@@ -162,7 +162,7 @@ pub mod pallet {
 
 			let _ = ensure_signed(origin)?;
 
-			let current_block_number = <frame_system::Module<T>>::block_number();
+			let current_block_number = <frame_system::Pallet<T>>::block_number();
```
Comment (chenzongxiong) [inline pallets/account-linker/src/lib.rs:165]: what's difference `Pallet` and `Module`? Just renaming?

Comment (geastwood) [reply pallets/account-linker/src/lib.rs:165]: same question here 👍

**Review by geastwood — COMMENTED** [review]
Here are only refactoring due to the polkadot upgrade?

**Comment by geastwood** [comment]
Let's have a small description when opening PR, it would really help to give others context.

**Review by h4n0 — APPROVED** [review]

File: pallets/account-linker/src/lib.rs (hunk 1 above)
Comment (Satoshi-Kusumoto) [reply pallets/account-linker/src/lib.rs:165]: At around v0.9.1, this is renamed from upstream

**Comment by Satoshi-Kusumoto** [comment]
> Here are only refactoring due to the polkadot upgrade?

Yes

> Let's have a small description when opening PR, it would really help to give others context.

Ok I will try to do my best. There wasn't one is because this is quite regular. Maybe I should have set the title to "bump deps to v0.9.4"

**Pull request merged**